postject-api.h

```diff
@@ -33,6 +34,11 @@ inline void postject_options_init(struct postject_options* options) {
   options->pe_resource_name = NULL;
 }
 
+static inline bool postject_has_resource() {
+  const char* sentinel = "POSTJECT_SENTINEL_fce680ab2cc467b6e072b8b5df1996b2:0";
+  return sentinel[strlen(sentinel) - 1] == '1';
+}
+
```

Comment:
This implementation will be optimized out by the compiler - `sentinel` is a local value to `project_has_resource` so the compiler could decide it's a constant false condition and not actually evaluate it at run-time.

Check it in godbolt.org: https://godbolt.org/z/rPKd1sEWe. If you you add `-O3` as a compiler option (top of the right panel), then you can see the generated code collapses to be nothing, since it is a constant false condition.

I think something like this could work (unfortunately can't use `strlen` on a volatile):

```C
static inline bool postject_has_resource() {
  static const volatile char* sentinel = "POSTJECT_SENTINEL_fce680ab2cc467b6e072b8b5df1996b2:0";
  return sentinel[51] == '1';
}
```